src/workflows/steps/conflictResolution.js

```diff
@@ -0,0 +1,155 @@
+import git from "../../git";
+import util from "../../utils";
+import logger from "better-console";
+import chalk from "chalk";
+import { remove } from "lodash";
+
+export function getLocalChanges( state ) {
+	const { dependencies } = state;
+
+	// creates an object of the changes you made to the package.json
+	// for the pre-releases.
+	const localChanges = dependencies.reduce( ( result, dep ) => {
+		result[ dep.pkg ] = "";
+		result[ dep.pkg ] = dep.version;
+		return result;
+	}, {} );
+
+	state.cr.localChanges = localChanges;
+}
+
+export function findConflictedPackageJSONChunks( state ) {
+	const { configPath, cr: { localChanges } } = state;
+	const contents = util.readFile( configPath );
+
+	const LEFT_MARKER = "<<<<<<<";
+	const RIGHT_MARKER = ">>>>>>>";
+	const MIDDLE_MARKER = "=======";
+
+	// Potentially ask git diff --check to have it return line numbers for diff markers.
+	// Would have to subtract one from each number to make with array.
+	const lines = contents.split( "\n" );
+
+	// Find the conflicted section start
+	// Save off previous line so we know where to insert
+	// Read out conflicted section into chunk array
+	// Remove conflicted section with splice
+	let conflictMarker = null;
+	const chunks = {};
+
+	const newLines = lines.reduce( ( memo, line, index ) => {
+		if ( line.includes( LEFT_MARKER ) ) {
+			conflictMarker = lines[ index - 1 ];
+			chunks[ conflictMarker ] = [];
+		} else if ( conflictMarker ) {
+			if ( line.includes( RIGHT_MARKER ) ) {
+				conflictMarker = null;
+			} else {
+				chunks[ conflictMarker ].push( line );
+			}
+		} else {
+			memo.push( line );
+		}
+
+		return memo;
+	}, [] );
+
+	Object.keys( chunks ).forEach( key => {
```

Comment:
This forEach has a lot going on inside that isn't exactly easy to determine when reading fresh. Let's try at least moving it out into one function if not multiple.